### PR TITLE
New Changes Made Now Looking Awesome (I swear😁😁)

### PR DESCRIPTION
**Pull request by CyberBoyAyush**
Changes Made:
➡ Added new text section (for username)
➡ Added new social button 
➡ fixed some icons
➡ added bg image (awesome one)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,52 +8,85 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/e5f29a6e0e.js" crossorigin="anonymous"></script>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" />
     <link rel="stylesheet" href="./style.css" />
     <title>Bhilai Hacks Links</title>
   </head>
   <body>
     <nav class="navbar navbar-dark bg-dark justify-content-center mb-3">
-      <a class="navbar-brand mb-0 h1" href="#">Bhilai Hacks</a>
+      <a class="navbar-brand mb-0 h1" href="#"><b>Bhilai Hacks</b></a>
```
Comment (praveenscience) [inline index.html:19]: Don't use deprecated `<b>` tags. Use `font-weight` in CSS.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Awesome stuff. Thanks, please check the changes requested and update the PR. 🔥

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -19,41 +21,68 @@
     <div class="container">
       <div class="row">
         <div class="col-lg-6 offset-lg-3 col-md-8 offset-md-2 col-sm-10 offset-sm-1">
-          <div class="text-center mb-3"><img class="logo" src="https://i.imgur.com/Ooa0irR.png" alt="Bhilai Hacks Logo" /></div>
+          <div class="text-center mb-3"><img class="logo" src="https://images2.imgbox.com/75/62/kXt4NOCn_o.png" alt="Bhilai Hacks Logo" /></div>
+          <h3><span><strong>@BhilaiHacks</strong></span></h3>
```
[the remark this answers is not in the document]
Comment (CyberBoyAyush) [reply index.html:25]: Fixed

File: index.html [hunk 3]
```diff
@@ -19,41 +21,68 @@
     <div class="container">
       <div class="row">
         <div class="col-lg-6 offset-lg-3 col-md-8 offset-md-2 col-sm-10 offset-sm-1">
-          <div class="text-center mb-3"><img class="logo" src="https://i.imgur.com/Ooa0irR.png" alt="Bhilai Hacks Logo" /></div>
+          <div class="text-center mb-3"><img class="logo" src="https://images2.imgbox.com/75/62/kXt4NOCn_o.png" alt="Bhilai Hacks Logo" /></div>
```
[the remark this answers is not in the document]
Comment (CyberBoyAyush) [reply index.html:24]: Done

File: index.html [hunk 4]
```diff
@@ -8,6 +8,8 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" />
     <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500&family=Open+Sans:wght@400;600&display=swap" rel="stylesheet" />
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/simple-line-icons/2.4.1/css/simple-line-icons.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/e5f29a6e0e.js" crossorigin="anonymous"></script>
```
[the remark this answers is not in the document]
Comment (praveenscience) [reply index.html:12]: Alright... So should you not remove the other one?

**Review by praveenscience — APPROVED** [review]
Even though it's not perfect, no one's perfect dude... 🔥

**Comment by CyberBoyAyush** [comment]
> Even though it's not perfect, no one's perfect dude... 🔥

Learning Bhai❤️❤️✌️✌️

**Pull request merged**